psse/psse-converter/src/main/java/com/powsybl/psse/converter/SwitchedShuntCompensatorConverter.java

```diff
@@ -186,6 +187,11 @@ private static int defineSectionCount(double binit, List<ShuntBlock> shuntBlocks
             }
         }
 
+        // Add the zero block, shunt disconnected
+        shuntBlocks.add(new ShuntBlock(1, 1, 0.0));
+
+        shuntBlocks.sort(Comparator.comparing(ShuntBlock::getB));
```

Comment:
Here, just to be sure: the sections are accumulated from the smaller b to the greater b in absolute value ?

Reply:
Yes, the sections are accumulated from the smaller b to the greater b in absolute value.